Microsoft.O365.Security.Native.ETW/Microsoft.O365.Security.Native.ETW.vcxproj

```diff
@@ -20,13 +20,13 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ProjectGuid>{ED4E6027-541F-440A-A5EE-15DBB7B89423}</ProjectGuid>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```

Comment:
Does this break back-compat for existing users who are installing the nuget on .NET versions <4.7.2?

Reply:
I honestly do not know. Not really familiar with .NET so unsure about the consequences. Unfortunately 4.5 is not available on the publicly available runners.

Reply:
Does the runner fail if we don't change the TargetFrameworkVersion (i.e. continue to target v4.5)? If so, could you try v4.5.2 which should be compatible with all future versions?

Reply:
Actually works without any .Net changes now.

Reply:
outstanding! thanks for debugging through this.